src/app/components/navigation/navigation.component.css

```diff
@@ -13,42 +19,50 @@
 }
 
 nav {
-  box-shadow: 0px 7px 5px -9px rgba(0,0,0,0.75);
-  -webkit-transition:0.5s linear all;
-  -moz-transition:0.5s linear all;
-  -o-transition:0.5s linear all;
-  transition:0.5s linear all;
+  box-shadow: 0px 0px 5px rgba(0,0,0,0.75);
+  -webkit-transition: 0.2s linear height;
+  -moz-transition: 0.2s linear height;
+  -o-transition: 0.2s linear height;
+  transition: 0.2s linear height;
+  height: 80px;
 }
 
 .github {
-  width: 100px;
-  height: 100px;
+  height: 100%;
 }
 
-@media(min-width:992px){
+@media(min-width:992px) {
   .shrink {
     height: 60px;
   }
 
-  .shrink .logo {
-    width: 200px;
-  }
 
   .shrink .github {
     width: 60px;
     height: 60px;
   }
 }
 
+@media(max-width: 992px) {
+  .logo {
+    height: 55px;
+  }
+}
+
 @media(max-width: 1300px) {
-	 .github {
-		         display: none
-			  }
-		  }
+  .github {
+    display: none
+  }
+
+  nav {
+    height: auto;
```

Comment:
It seems that a fixed height is used everywhere else. Why using an `auto` height here? TBH I am working on a information banner and I need the nav height to be always fixed with a known value. Would it be possible to let the nav height be 80px here as well?

Reply:
>  Would it be possible to let the nav height be 80px here as well?

It is not possible to fix the nav height on devices smaller than 992px because the navigation bar switch to a drop down menu. The height is then variable in that case, hence making it auto to not break the menu.

![Screenshot 2021-08-25 at 16-12-31 Zonemaster](https://user-images.githubusercontent.com/19394895/130806695-824f5032-ae9e-43df-b7f6-f998eeddf6ab.png)

If you need to know the height of the navigation bar, this PR adds a new service `NavigationService` because i had the same problem. This service exposes a observable that you can subscribe to to update a fixed height somewhere.

Relevant files:
https://github.com/zonemaster/zonemaster-gui/blob/e246b4571079577c311e1a6efeb15da1d89da189/src/app/services/navigation.service.ts#L4-L10

https://github.com/zonemaster/zonemaster-gui/blob/e246b4571079577c311e1a6efeb15da1d89da189/src/app/components/result/result.component.ts#L89-L91

https://github.com/zonemaster/zonemaster-gui/blob/e246b4571079577c311e1a6efeb15da1d89da189/src/app/components/result/result.component.html#L159